docs/references/message.md

```diff
@@ -39,14 +52,14 @@ title: Message
     * [.recall\(\)](contact.md#Message+recall) ⇒ `bool`
   * _静态方法_
     * [.find\(\)](message.md#Message.find) ⇒ `Optional[Message]`
-    * [.find_all\(\)](message.md#Message.findAll) ⇒ `List[Message]`
+    * [.find_all\(\)](message.md#Message.findAll) ⇒ `List[Message]` -->
     
 
-### ~~def from\(self\)~~ ⇒ `Contact`
+<!-- ### ~~def from\(self\)~~ ⇒ `Contact`
 
-已弃用, 详见[message.talker\(\)](message.md#Message+talker)
+已弃用, 详见[message.talker\(\)](message.md#Message+talker) -->
 
-### def talker\(self\) ⇒ `Contact`
+<!-- ### def talker\(self\) ⇒ `Contact`
 
```

Comment:
This desc style is from source of typescript, we should change it to mkdocs style, the same as the api module.   How do you think about it? 

this desc looks weired.

Reply:
Yes, I found that many of the documentation descriptions were lifted directly from the typescript section, and these codes should be rewritten by python.
And I will rewrite the corresponding section later when I write this part of the document.